iree/compiler/Codegen/Common/ShapeToLLVMConversion.cpp

```diff
@@ -1,104 +0,0 @@
-// Copyright 2021 The IREE Authors
```

Comment:
Wondering why this file was deleted. Does this completely drop shape dialect and therefore these patterns are no more needed?

Reply:
Yup. `MaterializeInterfaces` controls what the backend and HAL executable translation pipelines see. `tie_shape` ops are now removed in `MaterializeInterfaces` so backends shouldn't see them at all. Removing this doesn't break anything . That's a sign of that. (I might miss something somewhere though given those shape stuff is everywhere. But I believe the leftover should be minor issues and we can clean up when they pop up. Removing these patterns also makes those reveal easily, as if they fall through compilation will fail.)